diff --git a/libraries/liblmdb/Makefile b/libraries/liblmdb/Makefile
index 9fd819b398..ee2be03e25 100644
--- a/libraries/liblmdb/Makefile
+++ b/libraries/liblmdb/Makefile
@@ -28,6 +28,7 @@ LDFLAGS = $(THREADS)
 LDLIBS	= 
 SOLIBS	= 
 SOEXT	= .so
+BINEXT  =
 LDL		= -ldl
 
 LIBVER	= 1
@@ -67,7 +68,7 @@ install: $(ILIBS) $(IPROGS) $(IHDRS)
 	mkdir -p $(DESTDIR)$(libdir)
 	mkdir -p $(DESTDIR)$(includedir)
 	mkdir -p $(DESTDIR)$(mandir)/man1
-	for f in $(IPROGS); do cp $$f $(DESTDIR)$(bindir); done
+	for f in $(IPROGS); do cp $$f$(BINEXT) $(DESTDIR)$(bindir); done
 	for f in $(ILIBS); do cp $$f $(DESTDIR)$(libdir); done
 	for f in $(ILIBS2); do cp $$f $(DESTDIR)$(libdir); \
 		i=`basename -s .$(ABIVER) $$f`; rm -f $(DESTDIR)$(libdir)/$$i; \
