Disable rpath for the bootstrap build with CMake.

diff --git a/Sources/Runtimes/CMakeLists.txt b/Sources/Runtimes/CMakeLists.txt
index f5c1e8a..33575f7 100644
--- a/Sources/Runtimes/CMakeLists.txt
+++ b/Sources/Runtimes/CMakeLists.txt
@@ -21,7 +21,6 @@ project(SwiftPMRuntime
 enable_language(C)
 include(GNUInstallDirs)
 
-set(CMAKE_INSTALL_NAME_DIR "@rpath")
 set(CMAKE_INSTALL_RPATH "$ORIGIN/../../$<LOWER_CASE:${CMAKE_SYSTEM_NAME}>")
 set(CMAKE_MSVC_RUNTIME_LIBRARY MultiThreadedDLL)
 set(CMAKE_POSITION_INDEPENDENT_CODE ${BUILD_SHARED_LIBS})
diff --git a/Sources/Runtimes/CompilerPluginSupport/CMakeLists.txt b/Sources/Runtimes/CompilerPluginSupport/CMakeLists.txt
index 95d3720..26e113a 100644
--- a/Sources/Runtimes/CompilerPluginSupport/CMakeLists.txt
+++ b/Sources/Runtimes/CompilerPluginSupport/CMakeLists.txt
@@ -12,8 +12,6 @@ target_link_libraries(CompilerPluginSupport PRIVATE
   PackageDescription)
 
 if(APPLE)
-  target_link_options(CompilerPluginSupport PRIVATE
-    "SHELL:-Xlinker -install_name -Xlinker @rpath/CompilerPluginSupport.dylib")
 else()
   target_link_libraries(CompilerPluginSupport PRIVATE
     Foundation)
diff --git a/Sources/Runtimes/PackageDescription/CMakeLists.txt b/Sources/Runtimes/PackageDescription/CMakeLists.txt
index efdcdb0..0fd4045 100644
--- a/Sources/Runtimes/PackageDescription/CMakeLists.txt
+++ b/Sources/Runtimes/PackageDescription/CMakeLists.txt
@@ -32,8 +32,6 @@ target_compile_definitions(PackageDescription PRIVATE
   USE_IMPL_ONLY_IMPORTS)
 
 if(APPLE)
-  target_link_options(PackageDescription PRIVATE
-    "SHELL:-Xlinker -install_name -Xlinker @rpath/libPackageDescription.dylib")
 else()
   target_link_libraries(PackageDescription PRIVATE
     Foundation)
diff --git a/Sources/Runtimes/PackagePlugin/CMakeLists.txt b/Sources/Runtimes/PackagePlugin/CMakeLists.txt
index d95d56b..611f9b7 100644
--- a/Sources/Runtimes/PackagePlugin/CMakeLists.txt
+++ b/Sources/Runtimes/PackagePlugin/CMakeLists.txt
@@ -22,8 +22,6 @@ add_library(PackagePlugin
   Utilities.swift)
 
 if(APPLE)
-  target_link_options(PackagePlugin PRIVATE
-    "SHELL:-Xlinker -install_name -Xlinker @rpath/libPackagePlugin.dylib")
 else()
   target_link_libraries(PackagePlugin PRIVATE
     Foundation)
